### PR TITLE
[CIRCSTORE-146] Add userId property to scheduled notice schema

### PR DESCRIPTION
**Pull request by dmytrotur**
## Purpose 
Add userId property to scheduled notice schema
Resolves: [CIRCSTORE-146](https://issues.folio.org/browse/CIRCSTORE-146)

### CODE REVIEW

File: ramls/scheduled-notice.json [hunk 1]
```diff
@@ -17,6 +17,11 @@
       "description": "UUID of related request for request notices",
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
     },
+    "userId": {
```
Comment (marcjohnson-kint) [inline ramls/scheduled-notice.json]: Could this be given a name that better reflects the role the user is playing.

When I read the comment, it sounds like this is the recipient user?

Comment (dmytrotur) [reply ramls/scheduled-notice.json]: Yes, makes sense for me. I'm thinking about "recipientId". What do you think, does it reflects the purpose of this field?

Comment (marcjohnson-kint) [reply ramls/scheduled-notice.json]: My personal preference would be `recipientUserId` as that describes both the role and the kind of record we expect the reference to refer to.

Comment (dmytrotur) [reply ramls/scheduled-notice.json]: Agree, changed to "recipientUserId"

**Pull request merged**